gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageRequestToAccessBoundaryConverter.java

```diff
@@ -46,8 +48,10 @@
       return translateObjectInsertRequest((Storage.Objects.Insert) request);
     } else if (request instanceof Storage.Objects.Compose) {
       return translateObjectComposeRequest((Storage.Objects.Compose) request);
-    } else if (request instanceof Storage.Objects.Get) {
```

Comment:
Should we fast-fail on plain `Get` request? Or show it will be treated if we accidentally will start to use it?

Reply:
If there already fast-fail for this, then it's OK.